engine/schema/src/main/resources/META-INF/db/schema-41710to41800.sql

```diff
@@ -586,3 +586,14 @@ UPDATE  cloud.configuration
 SET     description = "Use SSL method used to encrypt copy traffic between zones. Also ensures that the certificate assigned to the zone is used when
 generating links for external access."
 WHERE   name = 'secstorage.encrypt.copy';
+
+-- allow isolated networks without services to be used as is.
+UPDATE networks ntwk
+  SET ntwk.state = 'Implemented'
+  WHERE ntwk.network_offering_id in
+    (SELECT id FROM network_offerings ntwkoff
```

Comment:
@weizhouapache , than also
```suggestion
    (SELECT id FROM `cloud`.`network_offerings` ntwkoff
```
?